app/models/course.rb

```diff
@@ -265,6 +265,7 @@ def training_module_ids
 
   def tracked_revisions
     revisions.where.not(article_id: articles_courses.not_tracked.pluck(:article_id))
+             .where('revisions.wiki_id in (?)', wiki_ids)
```

Comment:
This should be done the more Rails-y way, `.where(wiki_id: wiki_ids)`

Reply:
Done!